.gitignore

```diff
@@ -8,4 +8,3 @@ gh-pages/
 karma.conf.js
 node_modules
 npm-debug.log*
-package-lock.json
```

Comment:
is this intentional?

Reply:
Yes. I explained it in https://github.com/aframevr/aframe/pull/5092#issuecomment-1216644163 
see documentation https://github.com/actions/setup-node#caching-global-packages-data